vhd/packer/configure-windows-vhd.ps1

```diff
@@ -85,7 +85,8 @@ function Get-FilesToCacheOnVHD
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.3/windowszip/v1.17.3-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.4/windowszip/v1.17.4-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.0/windowszip/v1.18.0-1int.zip",

```

Comment:
get rid of 1.18.0?

Reply:
Let me just double-check with @marosset, because it appeared to me we've been keeping three patches per release in the Windows VHD. If that was unintended, I'll remove this.

Reply:
Since the Windows VHD isn't space-constrained, and this image is shared with AKS, it behooves us to keep more patch versions around. We'll make sure @marrosset checks such changes to the Windows VHD.

Reply:
should be fine to keep